Sources/Jetworking/Client/Client.swift

```diff
@@ -12,8 +12,9 @@ public enum APIError: Error {
 }
 
 public final class Client {
+    public typealias RequestResult<ResponseType> = (HTTPURLResponse?, Result<ResponseType, Error>)
```

Comment:
What is the difference to `RequestCompletion`? Looks like you have the same type signature for it.

Reply:
The `RequestCompletion` is a completion function but the `RequestResult` is simply a return value. I would like to keep it as is to be more explicit. Furthermore I restructured the code so these type aliases are in the correct place.